tests/batch_test.py

```diff
@@ -108,6 +108,7 @@ def batch_request(spy_plugin_controller, test_data, request):
     return BatchRequest(
         'http://localhost:8888',
         spy_plugin_controller,
+        pyramid_request=mock.Mock(),
```

Comment:
possible to pass in a Request object rather than a mock? `pyramid.request.Request.blank('/')` maybe?

Reply:
using `self.mock_request` that has already been set up :)